onnxruntime/contrib_ops/cuda/bert/bert_padding.cu

```diff
@@ -361,6 +362,57 @@ void LaunchTrtSequenceOffset(int* trt_mha_padding_offset,
   }
 }
 
+__global__ void __launch_bounds__(kMAX_THREADS_PER_BLOCK)
```

Comment:
Add a comment that this kernel assumes left side or right-side padding. It does not support mask with more than 2 segments of ones and zeros like [1, 1, 0, 0, 1, 1] or [0, 0, 1, 1, 0, 0]

Reply:
Let's use different checkin for the test cases.